src/firewall/core/rich.py

```diff
@@ -417,6 +424,15 @@ def _import_from_string(self, rule_str):
                     in_elements.pop() # protocol
                 else:
                     raise FirewallError(errors.INVALID_RULE, "invalid 'protocol' element")
+            elif in_element == 'tcp-mss-clamp':
+                if attr_name == 'value':
+                    self.element = Rich_Tcp_Mss_Clamp(attr_value)
+                    in_elements.pop()
```

Comment:
It's unnecessary to create the `Rich_Tcp_Mss_Clamp` instance here. Check other examples.. they set `attrs` which will be used later in the `else` block.